app/controllers/email_alert_subscriptions_controller.rb

```diff
@@ -5,16 +5,23 @@ class EmailAlertSubscriptionsController < ApplicationController
   helper_method :subscriber_list_params
 
   def create
-    if valid_choices?
-      redirect_to email_alert_signup_api.signup_url
-    else
-      @error_message = "Please choose an email alert"
-      render action: :new
-    end
+    validate_choices!
+    redirect_to email_alert_signup_api.signup_url
+  rescue MissingFiltersError
+    render_error "Please choose an email alert"
+  rescue EmailAlertSignupAPI::UnprocessableSubscriberListError
```

Comment:
Should the `StandardError` case have been removed?

Reply:
Yeah, I think I'd prefer returning a 500 than a 200 in this case.